shared/build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
     kotlin("plugin.serialization")
     id("com.android.library")
     id("kotlin-android-extensions")
+    id("com.chromaticnoise.multiplatform-swiftpackage") version "1.0.2"
```

Comment:
This is the magic that lets us use SPM with a Kotlin framework, it's a third party gradle plugin that converts the build iOS framework into an XCFramework which is what SPM needs

Reply:
So cool

Reply:
![Alt Text](https://media.giphy.com/media/12NUbkX6p4xOO4/source.gif)